### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by ismvru**
Add docker container build instructions

How to build:

```bash
docker build . --tag SomeTag:Version  #For example: pgtune/pgtune:latest
```

### CODE REVIEW

**Comment by le0pard** [comment]
Thanks @ismvru What is use case for this container?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM alpine:3 AS builder
```
Comment (le0pard) [inline Dockerfile:1]: Better start from `ruby` container

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+WORKDIR /code
+COPY . /code/
+RUN apk --no-cache add nodejs ruby-dev build-base git yarn \
+  &&  gem install --no-document middleman
```
Comment (le0pard) [inline Dockerfile:5]: middleman installed by bundler

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+RUN yarn install --immutable \
+  && yarn run lint \
```
Comment (le0pard) [inline Dockerfile:7]: is this just for build, you don't need `lint` and `test`

**Comment by le0pard** [comment]
If you just need builded files you can download them from here - https://github.com/le0pard/pgtune/archive/refs/heads/gh-pages.zip

**Comment by ismvru** [comment]
> Thanks @ismvru What is use case for this container?

You can raise a web service in your closed corporate network, from where, for example, there is no Internet access

**Comment by le0pard** [comment]
> You can raise a web service in your closed corporate network, from where, for example, there is no Internet access

in this case it can be just:

```
wget https://github.com/le0pard/pgtune/archive/refs/heads/gh-pages.zip
unzip gh-pages.zip
```

inside `nginx:alpine` container

**Comment by ismvru** [comment]
> > You can raise a web service in your closed corporate network, from where, for example, there is no Internet access
> 
> in this case it can be just:
> 
> ```
> wget https://github.com/le0pard/pgtune/archive/refs/heads/gh-pages.zip
> unzip gh-pages.zip
> ```
> 
> inside `nginx:alpine` container

Already did so in commit 9c8f130

File: Dockerfile (hunk 1 above)
Comment (ismvru) [reply Dockerfile]: Removed build part from container in 9c8f130

File: Dockerfile (hunk 3 above)
Comment (ismvru) [reply Dockerfile]: Removed build part from container in 9c8f130

**Pull request merged**